### PR TITLE
fix scan vertex/edge do not handle ttl

### PR DESCRIPTION
**Pull request by critical27**
## What type of PR is this?
- [X] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
https://github.com/vesoft-inc/nebula-ent/issues/1162

#### Description:
As title, scan vertex/edge does not handle ttl, and some of code in the community/ent version are not unified.

## Checklist:
Tests:
- [X] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/storage/exec/ScanNode.h [hunk 1]
```diff
@@ -262,9 +274,16 @@ class ScanEdgePropNode : public QueryNode<Cursor> {
     return nebula::cpp2::ErrorCode::SUCCEEDED;
   }
 
-  void collectOneRow(bool isIntId, std::size_t vIdLen) {
+  nebula::cpp2::ErrorCode collectOneRow(bool isIntId, std::size_t vIdLen) {
     List row;
     nebula::cpp2::ErrorCode ret = nebula::cpp2::ErrorCode::SUCCEEDED;
+    // Usually there is only one edge node, when all of the egdeNodes are invalid (e.g. ttl
+    // expired), just skip the row. If we don't skip it, there will be a whole line of empty value.
```
Comment (Shylock-Hg) [inline src/storage/exec/ScanNode.h:281]: In fact, we should return one only if vertex key exists.

Comment (critical27) [reply src/storage/exec/ScanNode.h:281]: Where did you handle this in scan before? BTW, the vertex key will be removed later

Comment (Shylock-Hg) [reply src/storage/exec/ScanNode.h:281]: When introducing this, there is no vertex key.

Comment (Shylock-Hg) [reply src/storage/exec/ScanNode.h:281]: It's a bug delayed https://github.com/vesoft-inc/nebula/issues/3688

**Review by pengweisong — APPROVED** [review]

File: src/storage/query/ScanVertexProcessor.cpp [hunk 2]
```diff
@@ -75,6 +75,7 @@ nebula::cpp2::ErrorCode ScanVertexProcessor::checkAndBuildContexts(
       return nullptr;
     }
   });
+  buildTagTTLInfo();
```
Comment (panda-sheep) [inline src/storage/query/ScanVertexProcessor.cpp:78]: 👍

**Review by panda-sheep — APPROVED** [review]
Good job

**Pull request merged**